### ISSUE TITLE
CoPilot integration

### ISSUE THREAD

**Issue by shahul01** [lukehaas/RunJS#414]
I know this is a huge ask, so maybe you can include this in premium version or something.

Since Github Copilot is available in other editors/IDEs, I'd appreciate if it could be made available in RunJS...

**Comment by kalon-robson** [comment]
+1

**Comment by zeckdude** [comment]
Yes! I complete agree with this request. Would be so amazing!

**Comment by CompaCodeRC** [comment]
It would be great if copilot could be added!

**Comment by devacran** [comment]
It will be awesome!

**Comment by MarlBurroW** [comment]
This is the only feature that is missing for me.
This feature will make this app a big must have.

**Comment by iqbalhussain931** [comment]
+1 This feature will be awesome.

**Comment by fzaffarana** [comment]
+1, this would be a game changer!

**Comment by lukehaas** [comment]
@shahul01 @kalon-robson @zeckdude @CompaCodeRC @devacran @MarlBurroW @fzaffarana RunJS now has OpenAI integration.

**Comment by shahul01** [comment]
Let's go!!

**Comment by jackmcpickle** [comment]
While this would be great, maybe an easy option now RunJS has chatGPT I think would be to add allow you to use your own AI server. Thinking specifically https://ollama.com/ - Would reduce the running costs of AI and enable users of lower means to access it. Zed editor has gone down that road.

**Comment by lukehaas** [comment]
@jackmcpickle I agree 👍 I'm looking into options such as ollama for a future update.
